tests/providers/aws/aws_e2e.py

```diff
@@ -118,14 +119,11 @@ def test_end_to_end_vm(self):
         volume_id=self.volume_to_forensic)
     self.volumes.append(volume_to_attach)
     # Create and start the analysis VM and attach the boot volume
-    self.analysis_vm, _ = forensics.StartAnalysisVm(
-        self.analysis_vm_name,
-        self.zone,
-        10,
-        4,
-        attach_volume=volume_to_attach,
-        device_name='/dev/sdp'
-    )
+    self.analysis_vm, _ = forensics.StartAnalysisVm(self.analysis_vm_name,
+                                                    self.zone,
+                                                    10,
+                                                    attach_volume=volume_to_attach,  # pylint: disable=line-too-long
+                                                    device_name='/dev/sdp')
```

Comment:
Just like in the comment above, you can use a 4-space hanging indent and skip the line-too-long